src/lib.rs

```diff
@@ -1,6 +1,15 @@
 //! This crate implements handling of proguard mapping files.
```

Comment:
_nit_: Since we're giving this an overhaul now, how would you feel about a bit more docs here and a checked example?